<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSettings</class>
 <widget class="QWidget" name="GlobalSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>15</number>
   </property>
   <item row="1" column="3">
    <widget class="QPushButton" name="songPathButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Urządzenie wejściowe do nagrywania</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pory zebrań</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Katalog z pieśniami</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLineEdit" name="recordInputEdit">
     <property name="toolTip">
      <string>Adres urządzenia używanego przez VLC (sprzawðź w odtwarzaczu)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Katalog do zapisywania nagrań</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QTimeEdit" name="ministerialTimeEdit"/>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Wykład publiczny i Studium Strażnicy</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QTimeEdit" name="publicTimeEdit"/>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="ministerialDayBox"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Studium Biblii, TSSK i Zebranie Służby</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLineEdit" name="recordPathEdit">
     <property name="toolTip">
      <string>Miejsce na dysku gdzie bedą zapisywane nagrania z zebrań</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QComboBox" name="publicDayBox"/>
   </item>
   <item row="13" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="songPathEdit">
     <property name="toolTip">
      <string>Miejsce na dysku z fortepianową wesją &quot;Śpiewajmy Jehowie&quot;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="recordPathButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
